app/views/recipes/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>I'm here to create a new recipe</h1>
+
+  <%= form_for :recipe, url: user_recipes_path  do |f| %>
+  <%= f.label :title %>
+  <%= f.text_field :title %>
+
+  <%= f.label :ingredients %>
+  <%= f.text_field :ingredients %>
+
+  <%= f.label :instructions %>
+  <%= f.text_field :instructions %>
+
+  <%= f.submit 'Create' %>
```

Comment:
You can use a bare `f.submit` here. If you passed an object to `form_for`, `f.submit` is smart enough to generate different text based on whether the object is already persisted or not to handle the create and update cases.